submarine-workbench/workbench-web/src/app/pages/workbench/model/model-version/model-version.component.html

```diff
@@ -16,20 +16,56 @@
   ~ specific language governing permissions and limitations
   ~ under the License.
   -->
-
 <div style="margin: 15px; padding: 15px; background-color: white">
-  <p>Version: {{ modelVersion }}</p>
-  <p>Description: {{ modelVersionInfo.description }}</p>
-  <p>Tags: {{ modelVersionInfo.tags }}</p>
-  <p>Stage: {{ modelVersionInfo.currentStage }}</p>
-  <h3>Model version info</h3>
-  <p>User id: {{ modelVersionInfo.userId }}</p>
-  <p>
-    Experiment id:
-    <a href="/workbench/experiment/info/{{modelVersionInfo.experimentId}}">{{ modelVersionInfo.experimentId }}</a>
-  </p>
-  <p>Dataset: {{ modelVersionInfo.dataset }}</p>
-  <p>Created: {{ modelVersionInfo.creationTime }}</p>
-  <p>Updated: {{ modelVersionInfo.lastUpdatedTime }}</p>
-  <p>Source: {{ modelVersionInfo.source }}</p>
-</div>
+  <nz-spin [nzSpinning]="isLoading">
+    <article nz-typography>
+      <h3 nz-typography>Version {{ modelVersion }}</h3>
+      <p nz-typography>
+        <span><strong>Description: </strong></span>
+        {{ isLoading ? null : modelVersionInfo.description }}
+      </p>
+      <p nz-typography>
+        <span><strong>Tags: </strong></span>
+        <span  *ngIf="!isLoading">
+          <submarine-model-tag *ngFor="let tag of modelVersionInfo.tags" [tag]="tag" [cssType]="'default'"></submarine-model-tag>
+        </span>
+      </p>
+      <p nz-typography>
+        <span><strong>Stage: </strong></span>
+        {{ isLoading ? null : modelVersionInfo.currentStage }}
+      </p>
+      <nz-divider [nzText]="text" nzOrientation="left">
+        <ng-template #text>
+          Model Version Info
+          <i nz-icon nzType="info-circle"></i>
+        </ng-template>
```

Comment:
Can you explain why adding this line?
I think all of the metadata is model version info.